cluster-up-example

```diff
@@ -1,14 +1,10 @@
 [Nodes:children]
 master
-core
-mbaas
+macos
```

Comment:
It looks like adding this is making `ansible_connection=local` have an effect on the macOS tasks, they're trying to run everything locally. Before `ansible_connection=local` was being ignored.